text/2020-03-11-versioned-kv.md

```diff
@@ -149,6 +149,10 @@ improve the performance of iterator seek. `key: ts-> delete` indicates that
 
 ### Version recycling
 
+The version recycling would recycling the redundant versions more than
+`MaxVerNum`, e.g. one key has five versions [v1 v2 v3 v4 v3], and the
+`MaxVerNum` is set to be 3, then [v1 v2] would be recycled.
```

Comment:
Feels like incremental backup will always lose some data if a key is updated very frequently no matter which consistency we choose. Is it okay for your case?

Reply:
it is okay for my case